docs/source/failure_modes.rst

```diff
@@ -90,6 +90,33 @@ Start the radar by:
 *NOTE* This may take a long time, and depends upon many factors including the antenna view of satellites, how long the
 octoclock-g has been powered off, temperature, etc. In testing it locked within 20 minutes.
 
+Front panel LEDS show incorrect pattern but Borealis continues to run normally
+------------------------------------------------------------------------------
+On several occasions, one of the N200 devices has shown a front panel LED pattern that doesn't match
+the expected pattern. The symptoms are:
+
+    - The IDLE (Yellow) or TXo (RED) LED(s) are lit when they should not be
+    - The N200 and Borealis system are operating normally
+
+Here is an example image from a webcam photo of the Saskatoon SuperDARN Borealis rack where the IDLE
+LED on N200 #4 is lit when it shouldn't be:
+
+.. image:: img/n200_LED_fault.jpg
+   :scale: 80%
+   :alt: N200 with IDLE LED incorrectly lit while Borealis operates normally
+   :align: center
+
+Fix this issue by:
```

Comment:
@kevinkrieger was the issue here a short? If so, where did the short occur?

Reply:
We're not sure actually, I just reseated the LED and it went away. Though it has come back now on the same box, it seems to be sporadic and only on one N200 out of the 42 we have deployed and 16 we have in the lab, so it's rare.